views/syntax.lavender

```diff
@@ -0,0 +1,58 @@
+
+extends layouts/default
+
+-current_page = "Syntax"
+
+block title
+  | Syntax View - Lavender Templating Engine for PHP
+
+block content
+  .container
+    .row
+      .col-md-8.col-md-offset-2.about.img-rounded
+          h1.title Make it look good
+          p
+            | So you're ready to write some code and you want it to look pretty?  Here's how it's done.
+  .container
+    .row
+      .col-md-8.col-md-offset-2.about.img-rounded
+        .col-md-5
+          img.img-responsive.center-block(src='/img/logos/sublime.png')          
+        .col-md-4.col-md-offset-2
+          h1 Sublime
+          p
+            | Since Lavender is based off of Jade, any text editor that can show Jade syntax will work.
+            | We prefer 
+            a(href='http://www.sublimetext.com/', target='_blank') <b>Sublime</b>
+            | but any editor that can read Jade will do.
+  .container
+    .row
+      .col-md-8.col-md-offset-2.about.img-rounded
+          h2 How it's done
+          p
+            | The first step is to install package control in Sublime
+            | follow the instructions for your preferred version of Sublime at
+            a(href='https://packagecontrol.io/installation', target='_blank') <b>Packagecontrol.io</b>
```

Comment:
`<b>` tags (along with `<i>` tags) are a thing of the past. You should use `<strong>` for bold and `<em>` for italics. The purpose for the new tags is so you can have your own definition of "emphasis"... but out of the box they do the same thing as the old tags.